### ISSUE TITLE
Suggestion: Chorus Fruit - End Rods

### ISSUE THREAD

**Issue by Azuriuz** [TeamMetallurgy/RTTS-Atum#75]
Can this be a rare loot table item? NEI says you can use it with Efreet Hearts to craft end rods, which are a very useful light source and deco item. It's like a torch, but you can place it on floors, walls, AND ceilings, and it is useful for alot of deco.

Chorus Fruit's other crafting recipe could either be removed or retextured for the pack. It's only other recipe is literally just PurPur Block, which could be changed to be cyan or something, maybe a deco block texture you never got to use or something because of item id limitations?

The fruit itself, when eaten, random teleports you. This feature could also maybe be disabled? Mostly it'd be nice to be able to craft end rods! So perhaps it could just be a change of recipe to something already in Atum?

**Comment by TopPercentageRattata** [comment]
Agreed, end rod recipe would be nice. Could give popped chorus fruits instead of uncooked ones if random teleportation in short directions would be an issue.

**Comment by Shadowclaimer** [comment]
Most likely I'd do a replacement texture and different recipe for the rods with a rename.

Maybe glowstone dust + limestone bits + cyan dye or something.
